curated_transformers/models/torchscript_wrapper.py

```diff
@@ -0,0 +1,148 @@
+from typing import Any, Callable, Optional, Union
```

Comment:
Would this module be a good candidate to add/move to `thinc`?

Reply:
Yeah, this should eventually go into Thinc, once we are happy with the class.

Reply:
I'll open an issue to keep track of it.